ariadne/src/Ariadne/Wallet/Backend/Tx.hs

```diff
@@ -142,20 +144,27 @@ sendTx acidDb WalletFace {..} CardanoFace {..} walletSelRef printAction pp walle
             signersMap = walletSigners esk wallets pp accountsToUse
         let getSigner :: Address -> Maybe SafeSigner
             getSigner addr = signersMap ^. at addr
-        -- TODO [AD-234]: generate new change address
         ourAddresses <-
             maybeThrow
                 (SendTxNoAddresses walletRootId)
                 (nonEmpty $ HM.keys signersMap)
-        let ourAddress = NE.head ourAddresses
+        -- We pick one of the accounts to generate change address in it.
+        ourAccount <-
+            maybeThrow
+                (SendTxNoAccounts walletRootId)
+                (toList accountsToUse ^? ix 0)
+        let ourAccountId = ourAccount ^. hdAccountId
+        let newChangeAddress = walletNewAddress
```

Comment:
Minor: redundant `let`, can be merged into the previous one.

Reply:
Some people say that such `let`s are redundant, while others insist on using explicit `let` for each binding. Life is hard.
:man_shrugging: 
Btw, [our style guide says](https://github.com/serokell/ariadne/blob/master/code-style.md#let-expressions):

> Put let before each variable inside a do block

I don't like it and want to make a PR removing this rule.